internal/pkg/cli/pipeline_init.go

```diff
@@ -181,6 +181,7 @@ func (o *initPipelineOpts) Execute() error {
 		}
 		log.Successf("Secret already exists for %s! Do nothing.\n", color.HighlightUserInput(o.GitHubRepo))
 	}
+	log.Successf("Created the secret %s for pipeline source stage\n", color.HighlightUserInput(secretName))
```

Comment:
nit: Can we end the log message with a punctuation? [STYLE_GUIDE#use full-sentences](https://github.com/aws/amazon-ecs-cli-v2/blob/master/STYLE_GUIDE.md#use-full-sentences)